anp/configs/radsecproxy/radsecproxy.conf

```diff
@@ -185,9 +185,17 @@ server OR-dynamic {
 	dynamicLookupCommand /etc/radsecproxy/naptr-openroaming.sh
 }
 
+server OR-dynamic-acct {
+	type tls
+	tls OR-client
+	secret radsec
+	certificateNameCheck off
+	dynamicLookupCommand "/etc/radsecproxy/naptr-openroaming.sh aaa+acct:radius.tls.tcp"
```

Comment:
No, the first parameter will be the realm, so radsecproxy will call naptr-openroaming.sh with '/etc/radsecproxy/naptr-openroaming.sh globalro.am', so I suggest you remove the NAPTR pattern.